CiE-iOS/CiE-iOS/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LoginViewController.swift
+//  CiE-iOS
+//
+//  Created by Jack Tan on 20/5/18.
+//  Copyright © 2018 Nikita Hans. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    @IBOutlet weak var usernameField: UITextField!
```

Comment:
> Don't link UIElements if you do not use them yet, try to implement this link once you actually need them, this is fine for now but in the future please do not link UIElement you do not use yet.

Reply:
Please use these UIElements to set their respective title, we in the future will need to handle multiple languages thats why we should assign the title of each button in the ViewController of that View -> we can than change it to appropriate titles for multiple languages later.

Reply:
Also set colors and such in the ViewController so that we can adjust or link all red color to one colorcode in our Stylerclass, you can not know that we will have such a thing but we should get one I will explain further in our next meeting.